mlrun/platforms/__init__.py

```diff
@@ -24,6 +25,48 @@
 )
 
 
+class _DeprecationHelper:
+    """A helper class to deprecate old schemas"""
+
+    def __init__(self, new_target: str, version="1.8.0"):
+        self._new_target = new_target
+        self._version = version
+
+    def __call__(self, *args, **kwargs):
+        self._warn()
+        return self._lazy_load()(*args, **kwargs)
+
+    def __getattr__(self, attr):
+        self._warn()
+        return getattr(self._lazy_load(), attr)
+
+    def _lazy_load(self, *args, **kwargs):
+        import mlrun.runtimes.mounts as mlrun_mounts
+
+        return getattr(mlrun_mounts, self._new_target)
+
+    def _warn(self):
+        warnings.warn(
+            f"mlrun.platforms.{self._new_target} is deprecated since version {self._version}, "
```

Comment:
We also specify when it will be removed (and add a `# TODO: remove in 1.10`)
```suggestion
            f"mlrun.platforms.{self._new_target} is deprecated since version {self._version}, and will be removed in 1.10"
```